src/Widgets/calendar/ControlHeader.vala

```diff
@@ -19,37 +19,64 @@
 
 
 namespace DateTime.Widgets {
-    public class ControlHeader : Gtk.Box {
+    public class ControlHeader : Gtk.Grid {
         public signal void left_clicked ();
         public signal void right_clicked ();
         public signal void center_clicked ();
+
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
+        }
+
+        construct {
+            var provider = new Gtk.CssProvider ();
+
+            var label = new Gtk.Label (new GLib.DateTime.now_local ().format (_("%OB, %Y")));
+            label.get_style_context ().add_class ("header-label");
+            label.xalign = 0;
+            label.width_chars = 13;
+
+            try {
+                provider.load_from_resource ("/io/elementary/desktop/wingpanel/datetime/main.css");
+                label.get_style_context ().add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```

Comment:
Instead of getting this style context twice, cache that:

```
            var label_style_context = label.get_style_context ();
            label_style_context.add_class ("header-label");
            label_style_context.add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```